tests/model/test_model.py

```diff
@@ -3745,6 +3745,20 @@ def test_get_next_unread_topic(
 
         assert unread_topic == next_unread_topic
 
+    @pytest.mark.parametrize(
+        "message_id, expected_value",
+        [
+            case(537286, (205, "Test"), id="message_in_topic_msg_ids"),
+            case(100000, None, id="message_not_in_topic_msg_ids"),
```

Comment:
With the current implementation I wouldn't expect much difference, but it might be worth including a known PM/DM message id in a test case?